src/DelegationToken.sol

```diff
@@ -30,13 +30,13 @@ abstract contract DelegationToken is IDelegation, ERC20PermitUpgradeable, Ownabl
         keccak256("Delegation(address delegatee,uint256 nonce,uint256 expiry)");
 
     // keccak256(abi.encode(uint256(keccak256("DelegationToken")) - 1)) & ~bytes32(uint256(0xff))
-    bytes32 internal constant ERC20DelegatesStorageLocation =
+    bytes32 internal constant DelegationTokenStorageLocation =
         0xd583ef41af40c9ecf9cd08176e1b50741710eaecf057b22e93a6b99fa47a6400;
 
     /* STORAGE LAYOUT */
 
     /// @custom:storage-location erc7201:DelegationToken
-    struct ERC20DelegatesStorage {
+    struct DelegationTokenStorage {
```

Comment:
As this storage is only about delegation
```suggestion
    struct DelegationStorage {
```

Reply:
It matches the name of the contract though.